tempto-examples/pom.xml

```diff
@@ -14,6 +14,7 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <main-class>io.trino.tempto.examples.TemptoExamples</main-class>
+        <air.test.skip>true</air.test.skip>
```

Comment:
add `<!-- there are no tests in this module -->`

Reply:
done